components/ShoppingList.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+
```

Comment:
You can import your categories and items here:

```suggestion
import initialCategories from "@/public/assets/categories.json";
import initialItems from "@/public/assets/shopping-items.json";
```